package.json

```diff
@@ -27,31 +27,17 @@
         "multi-root ready"
     ],
     "preview": true,
-    "activationEvents": [
```

Comment:
What's the minimum VSCode version for this pattern for activation events? Might need to update here.

Reply:
It was introduced in 1.74, the latest update Are we ok to require the latest? Or should I revert this change?

Reply:
I'm OK either way